### ISSUE TITLE
n_adj

### ISSUE THREAD

**Issue by dalanmaobu** [zju3dv/snake#162]
Hello author, I am not quite sure about the parameter ‘n_adj’ when I read the code, could you please explain it？
kernel_size=self.n_adj*2+1，what does it mean?

`def __init__(self, state_dim, out_state_dim=None, n_adj=4, dilation=1):
        super(DilatedCircConv, self).__init__()

        self.n_adj = n_adj
        self.dilation = dilation
        out_state_dim = state_dim if out_state_dim is None else out_state_dim
        self.fc = nn.Conv1d(state_dim, out_state_dim, kernel_size=self.n_adj*2+1, dilation=self.dilation)
`

**Comment by dalanmaobu** [comment]
Hello author, I am not quite sure about the parameter‘n_adj ’when I read the code, could you please explain it？There was something wrong with the editing of the last article
`class DilatedCircConv(nn.Module):
    def __init__(self, state_dim, out_state_dim=None, n_adj=4, dilation=1):
        super(DilatedCircConv, self).__init__()

        self.n_adj = n_adj
        self.dilation = dilation
        out_state_dim = state_dim if out_state_dim is None else out_state_dim
        self.fc = nn.Conv1d(state_dim, out_state_dim, kernel_size=self.n_adj*2+1, dilation=self.dilation)

    def forward(self, input, adj):
        if self.n_adj != 0:
            input = torch.cat([input[..., -self.n_adj*self.dilation:], input, input[..., :self.n_adj*self.dilation]], dim=2)
        return self.fc(input)`

**Comment by pengsida** [comment]
I do not us n_adj.
It is used in graph convolution.
